### DISCUSSION TITLE
Release artifacts

### DISCUSSION THREAD

**Discussion by DeltaCopy** [Bali10050/Darkly#13] [Ideas]
Is there a plan to add release artifacts ?

For Fedora we have a copr build working which generates a .rpm file.
https://copr.fedorainfracloud.org/coprs/deltacopy/lightly-qt6/

For Arch we now have a PKGBUILD which generates a .zst file

**Comment by DeltaCopy** [comment]
@Bali10050 the package is now in the AUR - [`lightly-qt6`](https://aur.archlinux.org/packages/lightly-qt6)
It currently builds from the Release tag [v0.5.1](https://github.com/Bali10050/Lightly/releases/tag/v0.5.1)

**Comment by Bali10050** [comment]
Thank you, I'll add something about it to the readme when I'm at home, and I'll make a new release to make sure everybody has the same version

**Comment by Bali10050** [comment]
Sorry for taking so long, I have added something about your works to the readme, 325cd89, I don't know if this was best way to do it, but atleast it's there now

**Reply by DeltaCopy** [reply]
Looks good and an alternative way to install is now documented.
I have updated both Fedora Copr and AUR with the latest release tag 0.5.2

Something I can look into is automating the build and packaging using Github Actions.
So whenever a release is tagged it'll automatically trigger a build from the tag,  package up in .rpm or .zst format and publish this as an asset.

**Reply by DeltaCopy** [reply]
https://github.com/Bali10050/Lightly/pull/27

Github Actions workflows:

![image](https://github.com/user-attachments/assets/ca0c9a4d-b66e-473f-9f3b-0a8c3ea5d846)

![image](https://github.com/user-attachments/assets/28cb5274-e2b1-4560-acef-24c6f9f31252)

**Comment by DeltaCopy** [comment]
Closing, since the latest release v0.5.3 show published assets now.

**Comment by Bali10050** [comment]
Thanks for helping!
